common/djangoapps/third_party_auth/tests/test_saml.py

```diff
@@ -0,0 +1,30 @@
+"""
```

Comment:
I had to move these tests out into a separate file because they were originally in the `SAMLTestCase` class of `common/djangoapps/third_party_auth/tests/testutil.py`. This class gets inherited from in `common/djangoapps/third_party_auth/tests/test_views.py`, so they would run multiple times. I'm not sure why this wasn't flagged when the older tests in the `test_views.py` file were added.